src/app/dataset-block/metadata-block/components/event-details/components/set-vocab-event/set-vocab-event.component.html

```diff
@@ -0,0 +1,29 @@
+<div *ngIf="event" class="position-relative border border-1 p-2 rounded-3">
+    <span class="position-absolute text-muted group-box-label text-uppercase"
+        >Information</span
+    >
+    <div class="mx-4" *ngIf="event.systemTimeColumn">
+        <app-block-row-data
+            [tooltip]="'Name of the system time column.'"
+            [label]="'System time column:'"
+        >
+            <span>{{ event.systemTimeColumn }}</span>
+        </app-block-row-data>
+    </div>
+    <div class="mx-4" *ngIf="event.eventTimeColumn">
+        <app-block-row-data
+            [tooltip]="'Name of the event time column.'"
+            [label]="'Event time column:'"
+        >
+            <span>{{ event.eventTimeColumn }}</span>
+        </app-block-row-data>
+    </div>
+    <div class="mx-4" *ngIf="event.offsetColumn">
+        <app-block-row-data
+            [tooltip]="'Name of the offset column.'"
+            [label]="'Offset column:'"
+        >
+            <span>{{ event.offsetColumn }}</span>
+        </app-block-row-data>
+    </div>
```

Comment:
This should look similar to what we defined in merge strategy:

![image](https://user-images.githubusercontent.com/8875909/217234288-946cec62-b3b9-4ec8-a02b-24dce971bfd3.png)

Reply:
Fixed